code/modules/client/preference_setup/global/preferences.dm

```diff
@@ -108,6 +108,14 @@ var/list/_client_preferences_by_type
 		sound_to(preference_mob, sound(null, repeat = 0, wait = 0, volume = 0, channel = GLOB.lobby_sound_channel))
 		sound_to(preference_mob, sound(null, repeat = 0, wait = 0, volume = 0, channel = GLOB.ambience_sound_channel))
 
+/datum/client_preference/play_jukeboxes
+	description = "Play jukeboxes"
+	key = "SOUND_JUKEBOXES"
+
+/datum/client_preference/play_boomboxes
+	description = "Play boomboxes"
+	key = "SOUND_BOOMBOXES"
```

Comment:
Seems excessive to have two preferences.
If someone another kind of player should that have a custom preference as well?

Reply:
We have boombox player which can used to play music. We can walkie-play with it and I thought what this is a good reason to add preference.

You may mute jukebox which probably can be only in bar, or you can mute boombox which probably can be anywhere.

Reply:
Just as an example: If someone adds a vehicle with a sound system, which may or may not be mobile, which of these two preferences should be used? Should a third be created just for that?

Reply:
Having one preference for music players is probably the way to go IMO, boomboxes and jukeboxes are pretty much the same conceptually.